libs/wire-api/src/Wire/API/MLS/Message.hs

```diff
@@ -160,6 +161,12 @@ data ContentType
 instance ParseMLS ContentType where
   parseMLS = parseMLSEnum @Word8 "content type"
 
+-- | Identify the content type of a message payload to be signed.
+mpTag :: MessagePayloadTBS -> ContentType
+mpTag (ApplicationMessage _) = ApplicationMessageTag
+mpTag (ProposalMessage _) = ProposalMessageTag
+mpTag (CommitMessage _) = CommitMessageTag
+
```

Comment:
Is this still used?

Reply:
In the refactored code it wasn't used. You just merged it so I guess we can remove it in some other PR.

Reply:
Yeah, I'm working on a follow up PR anyway, it's quicker to just merge as is rather than waiting for CI again and having to deal with stacked PRs in the meantime.